uwblueprint/paramedics-react
# Add transport flow

## Overview
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

[Notion task](https://www.notion.so/uwblueprintexecs/Change-Status-Transport-flow-36737be85ca640709ad2109a5ca0ecd8)

**To do**: 
- Link Patient Transported Toasts' "View Patient Details" button to patient Details modal on ccp dashboard page

## Changes
- Updates patient route
- Adds transport flow (transport confirmation modal, transport page, transport information fields on patient screen)
- updates patient queries/mutations to include assigned hospital and ambulance 
- Adds edit and delete patient options to ccp dashboard page, adds delete patient confirmation modal 

## Testing
- Transport flow can be accessed from Add/Edit patient screens
  - Update patient to transport status, assign patient hospital and ambulance
  - Make sure transport updates are reflected in ccp dashboard, patient table and on patient screen, and in db
- Delete a patient, make sure changes are reflected in patient table and in db
- Check edit patient option redirects to the correct patient screen
- Make sure cancel buttons in transport flow redirect to correct screen

## Screenshots
![image](https://user-images.githubusercontent.com/38408276/90340998-df89ec00-dfc9-11ea-97e0-495dcf8911a1.png)
![image](https://user-images.githubusercontent.com/38408276/90341000-e7e22700-dfc9-11ea-8821-bba1445b6d8a.png)
![image](https://user-images.githubusercontent.com/38408276/90341001-ec0e4480-dfc9-11ea-9496-865b16c46679.png)
![image](https://user-images.githubusercontent.com/38408276/90341006-f0d2f880-dfc9-11ea-9365-fbe6d4d37fa5.png)
![image](https://user-images.githubusercontent.com/38408276/90341014-f8929d00-dfc9-11ea-9382-6ffc4d212221.png)
![image](https://user-images.githubusercontent.com/38408276/90341024-034d3200-dfca-11ea-9e1d-149285541732.png)
![image](https://user-images.githubusercontent.com/38408276/90341029-0e07c700-dfca-11ea-950e-08494a5518ae.png)
